src/BizHawk.Emulation.Cores/Consoles/Nintendo/Mupen64/Mupen64.cs

```diff
@@ -78,7 +79,7 @@ public Mupen64(CoreLoadParameters<object, SyncSettings> loadParameters)
 		_debuggerUpdateCallback = DebuggerUpdateCallback;
 		Mupen64Api.DebugSetCallbacks(_debuggerInitCallback, _debuggerUpdateCallback, null);
 
-		ThrowIfError(Mupen64Api.CoreStartup(FRONTEND_API_VERSION, null, null, IntPtr.Zero, _debugCallback, IntPtr.Zero, _stateCallback), "CoreStartup failed");
+		ThrowIfError(Mupen64Api.CoreStartup(FRONTEND_API_VERSION, null, Path.Combine(AppContext.BaseDirectory, "dll"), IntPtr.Zero, _debugCallback, IntPtr.Zero, _stateCallback), "CoreStartup failed");
```

Comment:
```suggestion
		ThrowIfError(Mupen64Api.CoreStartup(FRONTEND_API_VERSION, null, PathUtils.DataDirectoryPath, IntPtr.Zero, _debugCallback, IntPtr.Zero, _stateCallback), "CoreStartup failed");
```

Reply:
I'd still need to combine that with `dll` though if I'm seeing it right. To be honest I'm not even sure if the .ini file should be with all other dlls but I also don't know a better place for it (gamedb?).